addons/compact-buttons/addon.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "Compact Buttons",
+    "description": "Buttons in projects get smaller/more compact",
```

Might want to further clarify this.
```suggestion
    "description": "Turns the text on some buttons on project and editor pages into icons to save space.",
```